contracts/Unipool.sol

```diff
@@ -26,22 +27,23 @@ contract LPTokenWrapper {
     function stake(uint256 amount) public {
         _totalSupply = _totalSupply.add(amount);
         _balances[msg.sender] = _balances[msg.sender].add(amount);
-        uni.safeTransferFrom(msg.sender, address(this), amount);
+        UNI.safeTransferFrom(msg.sender, address(this), amount);
     }
 
     function withdraw(uint256 amount) public {
         _totalSupply = _totalSupply.sub(amount);
         _balances[msg.sender] = _balances[msg.sender].sub(amount);
-        uni.safeTransfer(msg.sender, amount);
+        UNI.safeTransfer(msg.sender, amount);
     }
 }
 
-contract Unipool is LPTokenWrapper, IRewardDistributionRecipient {
-    IERC20 public snx = IERC20(0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F);
-    uint256 public constant DURATION = 7 days;
 
-    uint256 public periodFinish = 0;
-    uint256 public rewardRate = 0;
+contract Unipool is LPTokenWrapper, ApproveAndCallFallBack {
+    IERC20 public ANT = IERC20(0x960b236A07cf122663c4303350609A66A7B288C0);
```

Comment:
Same here, I'd move it after the constant since it is actually a storage variable